dcpy/utils/data.py

```diff
@@ -11,6 +12,21 @@
 import zipfile
 
 
+def _get_dtype(dtype: str | dict | None) -> str | dict | defaultdict | None:
```

Comment:
didn't end up using the defaultdict feature outside of a little testing but this seemed nice. When specifying a csv you can do
```yml
file_format:
    type: csv
    dtype:
        geoid: str
        __default__: float
```
To specify dtype for one column and a default for the rest

Reply:
Given your code snippet above, geoid column will be casted to a `string` and the rest will be casted to a `float`, correct?

Reply:
Precisely!